### DISCUSSION TITLE
Request documentation for adding Ignite to an existing project (if possible)

### DISCUSSION THREAD

**Discussion by prometheas** [infinitered/ignite#1734] [Q&A] [answered]
I have an existing project that I think would benefit from Ignite's capabilities. The README only seems to show a path for starting a project from scratch, so I'm left unsure as to whether adding the tool an existing project is even supported. Would love some explicit mention of the scenario and any potential approaches (supported or otherwise), if possible.

**Comment by jamonholmgren** [comment, marked as answer]
@prometheas I appreciate you asking! Infinite Red hasn't really done projects where we're adding Ignite to an existing project, so it hasn't been top of mind for us. What you might try is looking at the boilerplate folder and taking cues from there, or, alternatively, spinning up a new Ignite project and moving over existing code into it. Good idea though!
